pkgs/development/python-modules/scipy/default.nix

```diff
@@ -1,34 +1,41 @@
 {
   lib,
   stdenv,
-  fetchFromGitHub,
-  fetchpatch,
   fetchurl,
   writeText,
-  xcbuild,
   python,
   buildPythonPackage,
```

Comment:
If you want arguments to be ordered here in the order they are used in the expression (with the exception of `lib`), I'd put `buildPythonPackage` after `lib`.

Reply:
Isn't `buildPythonPackage` currently after `lib` ?

Reply:
> Isn't `buildPythonPackage` currently after `lib` ?

No :). Maybe you forgot to push something.

Reply:
Hmmm

![image](https://github.com/user-attachments/assets/28efbff6-a6e9-4106-9dd8-c89069d2ceea)

Reply:
Oh it is _after_ `lib`, but I meant it should be _right_ after `lib` - according to the rational.

Reply:
Ok I see.
Actually, `fetchurl`, `writeText` and `python` are used within the `let ... in` statement and thus _before_ `buildPythonPackage`. This is why I adopted this order.

Reply:
> Ok I see. Actually, `fetchurl`, `writeText` and `python` are used within the `let ... in` statement and thus _before_ `buildPythonPackage`. This is why I adopted this order.

Haha OK I forgot about that - I'll let you win again ;).